app/models/course.rb

```diff
@@ -323,15 +314,69 @@ class Course < ActiveHash::Base
             'You are an intermediate-level developer',
             'You might be familiar with version control and basic web architecture'
           ]
+        }
+      ]
+    }, {
+      id: 8,
```

Comment:
This is not only a translation of the course description into Portuguese: the course itself exists in Portuguese. In fact, there is a workshop in Brazil that is going to use it and it's almost ready to be scheduled.